example/src/app.js

```diff
@@ -21,7 +21,5 @@ export const App = (props, { state, effects }) => {
   );
 };
 
-App.contextTypes = contextTypes;
 
-
-export default addState(App);
+export default addState(injectState(App));
```

Comment:
I think at least the naming is confusing. While `react-intl` for example calls its `injectState` equivalent `injectIntl`, React Router calls it `withRouter`. As a user I'd mix these up.